app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt

```diff
@@ -454,6 +474,8 @@ class HomeActivityTest {
     }
 
     fun getUserAppHistoryController(): UserAppHistoryController
```

Comment:
Remove the usage of userAppHistory and its controller

Reply:
removed.